.github/workflows/test_examples_small.yml

```diff
@@ -9,12 +9,18 @@ env:
 jobs:
   test:
     runs-on: ${{ matrix.platform }}
-    name: "${{ matrix.platform }} ${{ matrix.toolchain }}"
+    name: "${{ matrix.example }} - ${{ matrix.platform }}"
     strategy:
+      fail-fast: false
       matrix:
         platform: [ubuntu-latest, macos-latest]
-        toolchain: [1.81]
-        # toolchain: [stable]
+        toolchain: [stable]
+        example: [
```

Comment:
splitting this into matrix is beneficial in terms of less storage space required to complete an individual chunk of work 
https://github.com/near/near-sdk-rs/actions/runs/11727834984/job/32669865173